src/components/views/settings/tabs/user/PreferencesUserSettingsTab.tsx

```diff
@@ -159,6 +159,7 @@ export default class PreferencesUserSettingsTab extends React.Component<IProps,
         'MessageComposerInput.ctrlEnterToSend',
         'MessageComposerInput.surroundWith',
         'MessageComposerInput.showStickersButton',
+        'MessageComposerInput.insertTrailingComma',
```

Comment:
>  I do have concerns about settings proliferating and causing maintenance burden.
>
> *-- @turt2live, https://github.com/matrix-org/matrix-react-sdk/pull/6711#pullrequestreview-744385080*

This feels like something we should just be opinionated on and avoid adding a setting. [Convention over configuration](https://about.gitlab.com/handbook/product/product-principles/#convention-over-configuration)

Reply:
If it were up to me, I would just remove this feature but I know some people care about having it quite a lot and therefore I've made it into a setting

Reply:
@MadLittleMods see vector-im/element-web#16682 for more context, there was some contention over this, and i think a good compromise to get this working at least one way or another is to make it into a setting, thus this PR.